homeassistant/components/sensor/deconz.py

```diff
@@ -35,7 +35,7 @@
     for sensor in sensors.values():
         if sensor and sensor.type in DECONZ_SENSOR:
             if sensor.type in DECONZ_REMOTE:
-                DeconzEvent(hass, sensor)
+                hass.data[DATA_DECONZ_EVENT].append(DeconzEvent(hass, sensor))
```

Comment:
@balloob do you want me to move the creation of the events and the class definition to be created by the setup entry method instead?

Reply:
Since it's just firing events, it should indeed be handled at the component level. It should not be done in the sensor platform.